exec/testing/fake_exec.go

```diff
@@ -240,6 +240,14 @@ func (fake *FakeCmd) Stop() {
 	// no-op
 }
 
+func (fake *FakeCmd) GetPath() string {
+	return fake.Argv[0]
```

Comment:
This would give the first arg which is the command name but does not do a realpath on it. Probably fine unless someone really needs to use this in a test scenario?